### PR TITLE
Translating functions to Bosnian

### PR DESCRIPTION
**Pull request by daddyDOT**
Translating functions to Bosnian progress...

### CODE REVIEW

**Comment by Southclaws** [comment]
Can we merge what's done so far and continue translations in a future PR?

**Comment by daddyDOT** [comment]
I will finish 'functions' translation by Saturday/Sunday, so if it's not a problem to wait just smth around 2 days. I will tag you when I upload everything.

File: docs/scripting/functions/PlayerTextDrawSetOutline.md [hunk 1]
```diff
@@ -1,46 +1,50 @@
 ---
 title: PlayerTextDrawSetOutline
-description: Set the outline of a player-textdraw.
+description: Postavite outline za player-textdraw.
```
Comment (Southclaws) [inline docs/scripting/functions/PlayerTextDrawSetOutline.md:3]: This is the English file

**Comment by Southclaws** [comment]
It seems a lot of these are in the top level English directory, right down to `docs/scripting/functions/valstr.md`

File: docs/scripting/functions/PlayerTextDrawSetOutline.md (hunk 1 above)
Comment (daddyDOT) [reply docs/scripting/functions/PlayerTextDrawSetOutline.md:3]: Ohhh shiit, my bad, I forgot about 'translations' directory

**Comment by daddyDOT** [comment]
Yeahh, I uploaded last 177 files in totally wrong path, -_______-

**Review by Southclaws — APPROVED** [review]

**Comment by Southclaws** [comment]
GitHub UI broke because there were so many changes and didn't show all the other changes. Rebase on master and make sure the only changes are in the docs directory. git filter-branch may help with this.

**Comment by daddyDOT** [comment]
I created second PR and pulled those files correctly and it seems that everything's fine, [click](https://github.com/openmultiplayer/web/pull/302)

**Pull request merged**